src/projects/republic/config/dutchRepublicLabels.ts

```diff
@@ -121,6 +121,8 @@ export const dutchRepublicLabels = Object.assign({}, dutchLabels, {
   NEXT: "Volgend resultaat",
   PREV: "Vorig resultaat",
   BACK_TO_SEARCH: "Terug naar resultaten",
-  
+
   OCCURRENCES_PER_YEAR: "Resoluties per jaar",
+
+  PROVENANCE: "View provenance",
```

Comment:
Dit is een Engels label in een NL bestand :)

Reply:
Fixed in [a5fc1fd](https://github.com/knaw-huc/textannoviz/pull/275/commits/a5fc1fdcf4b86856735e42f52a87366d0209d0f4)